src/Model/Resolver/Product/BundleProductOptions.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace ScandiPWA\CatalogGraphQl\Model\Resolver\Product;
+
+use Magento\Bundle\Model\Product\Price;
+use Magento\Bundle\Model\Product\Type as Bundle;
+use Magento\Catalog\Helper\Data as CatalogData;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Framework\GraphQl\Query\EnumLookup;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\Pricing\PriceCurrencyInterface;
+
+/**
+ * @inheritdoc
+ */
+class BundleProductOptions implements ResolverInterface
+{
+    /**
+     * Catalog data
+     *
+     * @var CatalogData
+     */
+    protected $catalogData;
+
+    /**
+     * @var EnumLookup
+     */
+    private $enumLookup;
+
+    /**
+     * @var PriceCurrencyInterface
+     */
+    private PriceCurrencyInterface $priceCurrency;
```

Comment:
Please don't use property types, since php 7.3 doesn't support it and it makes deploy to fail on some projects that haven't updated their php.